feat: adding extension files back in and readmes for each template (#40)
apollographql/rover-init-starters: start-with-rest/README.md

# Introduction 

👋 Welcome\!

This graph was created using `rover init`. 

> This template repository is used to initialize an API orchestration project using [Apollo’s Rover CLI](https://www.apollographql.com/docs/rover), [Apollo Connectors](https://www.apollographql.com/docs/graphos/schema-design/connectors), and an example REST API, by providing a structured setup for managing and composing APIs efficiently.

Reviewer: ```suggestion
`rover init` helps you set up a new, federated GraphQL API using [Apollo Connectors](https://www.apollographql.com/docs/graphos/schema-design/connectors) and an example REST API.
```

This command:

* Creates a set of template files in your current directory
* Generates the credentials your GraphQL API will need to interact with Apollo’s GraphOS platform
* Provides a starter schema to help you design your GraphQL API
* Walks you through spinning up a local environment to test your API

> Head over to \`getting-started.md\` for next steps.

Reviewer: Since this is stated in `## Next steps` you don't need it here.

```suggestion
```

> 📓 **Note:** If you’re already logged into an existing Apollo organization by the time you run this command, the CLI will nudge you to go to Apollo Studio. From there, you can create a new personal API key to interact with Rover. If you don’t have an account yet, visiting the link will prompt you to create one.

Reviewer: We refer to this as "GraphOS Studio" in the docs and the `getting.started.md` files

```suggestion
📓 **Note:** If you’re already logged into an existing Apollo organization by the time you run this command, the CLI will nudge you to go to GraphOS Studio. From there, you can create a new personal API key to interact with Rover. If you don’t have an account yet, visiting the link will prompt you to create one.
```
